app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```

Comment:
This line attempts to store the `Person` instance in the `people` dictionary using the person's name as the key. Ensure that names are unique, as this will overwrite any existing entry with the same name.